sao.js

```diff
@@ -107,6 +107,17 @@ module.exports = {
           ? true
           : 'Please include a valid GitHub.com URL without a trailing slash';
       }
+    },
+    trimWhitespaceMarkdown: {
+      message: 'Trim trailing whitespace in markdown files?',
+      type: 'confirm',
+      default: true
+    },
+    editorconfigIndentation: {
+      message: 'Indentation',
+      type: 'list',
+      choices: ['2 spaces', '4 spaces', 'tabs'],
+      default: '2 spaces'
```

Comment:
Curious if people have a global `~/.editorconfig` or something?  Not sure if htat's a standard, if so we could make default whatever value they use globally?

Reply:
I'm not personally aware of such a practice. I think that would assume there be tooling that looks for global `.editorconfig` files which could break the paradigm of configuring editors per project (maybe).

Reply:
Yeah, lets not worry about it